Bluejay.asm

```diff
@@ -152,6 +151,8 @@ DEFAULT_PGM_BEACON_STRENGTH		EQU	80	; 0..255
 DEFAULT_PGM_BEACON_DELAY			EQU	4	; 1=1m		2=2m			3=5m			4=10m		5=Infinite
 DEFAULT_PGM_ENABLE_TEMP_PROT		EQU	7	; 0=Disabled	1=80C	2=90C	3=100C	4=110C	5=120C	6=130C	7=140C
 
+DEFAULT_PGM_POWER_RATING		EQU	1	; 1=1S, 2=2S+
```

Comment:
I agree a checkbox is the way to go, but there are some restrictions for it. In bb51 during tests I think to remember that the internal 1.65v reference worked fine for 1s and 2s, but for bb21 it depends on the number of cells used.
I mean that I think that the checkbox should only appear for bb21 targets (in my opinion disabled by default) and though being occult for bb51 it should be forcibly checked. I also think that it should have value 1 when checked and 2 when unchecked, value 2 can be replaced by anyother though.

Reply:
Hmm, such a check would be tricky - at least I would need to add another layer apart from the `LAYOUT_REVISION`. For sake of consistency I would display it no matter if H or X. If X then the setting simply does nothing if checked (which I think is how it is currently implemented anyway).

If we want to use the checkbox widget, the values should really be 0 and 1, otherwise we will have to use a different widget type. Why would you want this checkbox to behave differently than all other checkboxes we are currently using? Maybe I am missing something. I can implement a different widget without big overhead, I just want to understand why we should handle it differently.

Reply:
Yeah, I agree that showing it anyways will be the best. I will also update the code so for BB51 the 3.3V vdd ADC reference code path won't run under any circumstances.